cs/django_installation/instructions.md

```diff
@@ -101,11 +101,11 @@ OK máme všechny důležité závislosti na místě a konečně můžeme nainst
 
 ## Instalace Django
 
-Máš své `virtualenv` nastartované a můžeš nainstalovat Django pomocí příkazu `pip`. V konzoli, spusť `pip install django == 1.8` (<0>Poznámka:</0> , používáme zde dvojité rovnítko: `==`).
+Máš své `virtualenv` nastartované a můžeš nainstalovat Django pomocí příkazu `pip`. V konzoli, spusť `pip install django == 1.10` (<0>Poznámka:</0> , používáme zde dvojité rovnítko: `==`).
 
 ```
-(myvenv) ~$ pip install django==1.8
-Downloading/unpacking django==1.8
+(myvenv) ~$ pip install django==1.10
```

Comment:
It looks like this locale was never updated to Django 1.9. The 1.8 -> 1.9 changes were less straightforward than 1.9 -> 1.10 - there were code changes with accompanying explanations which would require the involvement of translators to bring them to that point. So I suggest we should not update `cs` and any others that are still on 1.8 in this work. :smile: